### DISCUSSION TITLE
Error with "Create_Games" in Train Model

### DISCUSSION THREAD

**Discussion by Nikken85** [kyleskom/NBA-Machine-Learning-Sports-Betting#407] [Q&A] [answered]
Hello everyone, for a few days now, I've been experiencing the following error during the Train Model process, when I run "python -m Create_Games" (after successfully completing "Get_Data" and "Get_Odds_Data").
I want to mention that I've always gone through this step without any issues, but for the past few days, it keeps giving me this error. 
I've also tried it on another PC, and the same error persists.

![immagine](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/55053287/2ce7627e-ce30-4387-99df-feb98f773622)

What's happening? Is anyone able to complete the Train Model successfully these days? Is the issue only happening to me? Can anyone help me out?
Thanks a lot in advance.

**Comment by Scandianis** [comment, marked as answer]
Solved here:

https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/issues/403

**Reply by Nikken85** [reply]
It works!! Thank you!!! 👍
